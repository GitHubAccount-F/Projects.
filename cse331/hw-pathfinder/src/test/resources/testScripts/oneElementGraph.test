#Tests the Dijkstra's algorithm with a one element graph

#Creates the graph
CreateGraph graph1

#Adds a node
AddNode graph1 one

#Finds a path between two different nodes
FindPath graph1 one two

#Finds a path to the same node
FindPath graph1 one one