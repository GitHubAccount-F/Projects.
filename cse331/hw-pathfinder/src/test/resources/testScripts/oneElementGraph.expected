#Tests the Dijkstra's algorithm with a one element graph

#Creates the graph
created graph graph1

#Adds a node
added node one to graph1

#Finds a path between two different nodes
unknown: two

#Finds a path to the same node
path from one to one:
total cost: 0.000